lib/texting_twilio.rb

```diff
@@ -0,0 +1,29 @@
+require 'twilio-ruby'
+require 'dotenv'
+Dotenv.load
+
+module TextingTwilio
+
+  def self.send_text
+
+    account_sid = ENV['ACCOUNT_SID']
+    auth_token = ENV['AUTH_TOKEN']
+    from = ENV['TWIL_PHONE_NUMBER']
+    to = ENV['MY_PHONE_NUMBER']
+
+    delivery_time = Time.new + (60*60)
```

Comment:
Rails/TimeZone: Do not use `Time.new` without zone. Use one of `Time.zone.now`, `Time.new.current`, `Time.new.in_time_zone`, `Time.new.utc`, `Time.new.getlocal`, `Time.new.iso8601`, `Time.new.jisx0301`, `Time.new.rfc3339`, `Time.new.to_i`, `Time.new.to_f` instead. (https://github.com/bbatsov/rails-style-guide#time, http://danilenko.org/2012/7/6/rails_timezones)